index.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const port = process.env.PORT
+const app = express();
+var router = express.Router()
+var cors = require('cors')
+const bodyParser = require('body-parser')
+
+app.use(cors())
+app.use(bodyParser.json())
+
+const { initializeUserbase } = require('./IntialiseDB/UserbaseDB.js');
+initializeUserbase()
+
+const cart = require("./Routes/cart.js");
+app.use("/cart", cart)
+
+
+const products = require("./Routes/Products.js")
+app.use("/products", products)
+
+const users = require("./Routes/users.js")
+app.use("/users", users)
+
+const createSession=require("./Routes/create-checkout-session")
```

Comment:
I have a suggestion, you can keep all the "require" statements at the top, It looks a bit cleaner